stable/dex-k8s-authenticator/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for dex-k8s-authenticator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: mintel/dex-k8s-authenticator
+  tag: latest
```

Comment:
please don't use latest tag ever, as it may be changed on any commit from author, without any tests from us.